docs/user/features/properties/material.rst

```diff
@@ -7,8 +7,8 @@ Material Based Properties
 * The following section goes through those Blender material settings and how they relate to corresponding nif blocks or attributes.
 * Unless otherwise stated your mesh needs to have a material.
 * The nif format only supports a single material per :class:`~pyffi.formats.nif.NifFormat.NiTriShape`.
-* A Mesh which contains mult-materials will be exported as multiple :class:`~pyffi.formats.nif.NifFormat.NiTriShape`.
-* Giving the Material an appropriate name helps distingush materials and encourages reuse, eg. Metal, Glass, plastic etc.
+* A Mesh which contains multiple -materials will be exported as multiple :class:`~PyFFI.formats.nif.NifFormat.NiTriShape`.
```

Comment:
should be lower case as this is a module import.

Reply:
Ah, missed that. I used Find and replace, I did go through most of the files fixing this, I seem to have missed this. Will fix in my next commit.